### ISSUE TITLE
[BUG] Range needs VoiceOver support for alternating values

### ISSUE THREAD

**Issue by chi-jyskebank-dk** [kirbydesign/designsystem#3776]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
VoiceOver is only able to read out the current value of a Range slider. It would make sense to be able to programatically specify the text that is read out, eg. the value + a unit, or perhaps more advanced requirements for alternating texts.

## Describe how to reproduce the bug
1. Go to StackBlitz: https://stackblitz.com/edit/stackblitz-starters-wxb9gweu?file=src%2Fmain.ts
2. Ensure kirby-range [pin]='false' 
3. When adjusting the Range slider, only the new value is read out loud

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
It would be much preferred if the component consuming the Range, could expose a function with logic that could report to the Range what was supposed to be read out by VoiceOver. Eg. "100.000 kr.", "30 shares" or "55.000 kr., higher than recommended".

## Add any screenshots
Sample screen recording, showing how the current [pinFormatter] is used to simulate the expected behaviour:
https://github.com/user-attachments/assets/254e0ad5-09b5-44ec-b6f4-213f1085e1d7

## Please complete the following information:
- OS: macOS 15.2 with VoiceOver
- Browser: Google Chrome

## Are there any additional context?
I suggest a new optional callback is offered by the Range component, eg. [ariaSliderAdjustCallback]="someFunction", which will allow the consuming component, to decide the text read out by VoiceOver, based on the current value of the slider. Very much like the [pinFormatter] works. 

In the StackBlitz mentioned above, set kirby-range [pin]='true', to get an idea of what kind of flexibility we wish for when using VoiceOver (think of the values shown in the Pin, as values that would instead be read out to the user). In this case it doesn't make sense to show the text in a pin, as it is rather long, but make perfect sense when being read out loud.  

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
